modAionImpl/test/org/aion/zero/impl/vm/InternalTransactionTest.java

```diff
@@ -287,18 +287,6 @@ public void testRecursiveCall() throws InterruptedException, VMException {
         AionTxExecSummary summary = exec.execute().get(0);
 
         assertEquals(2, summary.getInternalTransactions().size());
-
```

Comment:
reason for removing the test section?

Reply:
The functions tested in this section are being deleted